dynamic_functions/constantsFunction.py

```diff
@@ -0,0 +1,14 @@
+def string(session_id, string):
+    return str(string)
+
+
+def integer(session_id, integer):
+    return int(integer)
+
+
+def float(session_id, float):
+    return float(float)
+
+
+def boolean(session_id, boolean):
```

Comment:
session_id not used

Reply:
> session_id not used

Yes, thats true but the session id must be included from all dynamic function. Due to the way how these functions are called.